bundles/org.openhab.binding.plugwiseha/src/main/java/org/openhab/binding/plugwiseha/internal/api/model/DTO/ActuatorFunctionalities.java

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
The folder name needs to be all lowercase.

Reply:
When it was lowercase, it started to warn about null annotations, maybe that has been fixed in the mean while, will test!

Reply:
fixed